squirrel.worker/src/test/resources/Sample_Files/sample.nt

```diff
@@ -0,0 +1,141 @@
+<http://dbpedia.org/resource/Orleans,_New_York> <http://www.w3.org/2002/07/owl#differentFrom> <http://dbpedia.org/resource/New_York> .
+<http://dbpedia.org/resource/George_Crum> <http://dbpedia.org/ontology/birthPlace> <http://dbpedia.org/resource/New_York> .
+<http://dbpedia.org/resource/George_Crum> <http://dbpedia.org/ontology/deathPlace> <http://dbpedia.org/resource/New_York> .
+<http://dbpedia.org/resource/Mary_Jo_Putney> <http://dbpedia.org/ontology/birthPlace> <http://dbpedia.org/resource/New_York> .
+<http://dbpedia.org/resource/Harold_Arlen> <http://dbpedia.org/ontology/birthPlace> <http://dbpedia.org/resource/New_York> .
+<http://dbpedia.org/resource/Port_Morris,_Bronx> <http://dbpedia.org/ontology/isPartOf> <http://dbpedia.org/resource/New_York> .
+<http://dbpedia.org/resource/Francis_W._Palmer> <http://dbpedia.org/ontology/region> <http://dbpedia.org/resource/New_York> .
+<http://dbpedia.org/resource/Halcott_Mountain> <http://dbpedia.org/ontology/locatedInArea> <http://dbpedia.org/resource/New_York> .
+<http://dbpedia.org/resource/Myzery> <http://dbpedia.org/ontology/hometown> <http://dbpedia.org/resource/New_York> .
+<http://dbpedia.org/resource/Soothe_My_Soul> <http://dbpedia.org/ontology/recordedIn> <http://dbpedia.org/resource/New_York> .
+<http://dbpedia.org/resource/Frank_Henry> <http://dbpedia.org/ontology/birthPlace> <http://dbpedia.org/resource/New_York> .
+<http://dbpedia.org/resource/Michael_Loew> <http://dbpedia.org/ontology/birthPlace> <http://dbpedia.org/resource/New_York> .
+<http://dbpedia.org/resource/Oscar_Shaw> <http://dbpedia.org/ontology/deathPlace> <http://dbpedia.org/resource/New_York> .
+<http://dbpedia.org/resource/E._G._Squier> <http://dbpedia.org/ontology/birthPlace> <http://dbpedia.org/resource/New_York> .
+<http://dbpedia.org/resource/Eduardo_Marquina> <http://dbpedia.org/ontology/deathPlace> <http://dbpedia.org/resource/New_York> .
+<http://dbpedia.org/resource/Iris_M._Ovshinsky> <http://dbpedia.org/ontology/birthPlace> <http://dbpedia.org/resource/New_York> .
+<http://dbpedia.org/resource/Maxine_Linehan> <http://dbpedia.org/ontology/residence> <http://dbpedia.org/resource/New_York> .
+<http://dbpedia.org/resource/Sid_Brod> <http://dbpedia.org/ontology/birthPlace> <http://dbpedia.org/resource/New_York> .
+<http://dbpedia.org/resource/Bohdan_Paw\u00C5\u0082owicz> <http://dbpedia.org/ontology/deathPlace> <http://dbpedia.org/resource/New_York> .
+<http://dbpedia.org/resource/Tonawanda_(town),_New_York> <http://www.w3.org/2002/07/owl#differentFrom> <http://dbpedia.org/resource/New_York> .
+<http://dbpedia.org/resource/Ethan_A._Hitchcock_(general)> <http://dbpedia.org/ontology/deathPlace> <http://dbpedia.org/resource/New_York> .
+<http://dbpedia.org/resource/Greg_Gutfeld> <http://dbpedia.org/ontology/residence> <http://dbpedia.org/resource/New_York> .
+<http://dbpedia.org/resource/BDMI> <http://dbpedia.org/ontology/location> <http://dbpedia.org/resource/New_York> .
```

Comment:
Test cases typically aim at certain situations. This looks like a simple copy from DBpedia. Although it does the job, it is better to a) reduce the number of triples to a number that a human can overlook fast and b) make sure that this small number covers all important cases (e.g., the usage of blank nodes or literals seems to be missing in this file).

Reply:
Nice. Could you please make sure that you have a test in which you have literals as objects? It looks like this important part is missing in the current test files.

Reply:
Do you been mean the files containing the test cases?

Reply:
Or the sample test files used as input to check the automata?

Reply:
Yes. Your current test files are focusing on URI resources as subject and object. Typically, you would also have examples with literals as objects and blank nodes as subjects and objects.
Unfortunately, these missing tests led to the problem described in the comment in the `FiniteStateMachineFactory.java` class.

Reply:
Ok, so the Automata's written are completely focused on triple which has the subject, object, and predicate in URI format. I focused completely on the resources which were available in the project.